cves/CVE-2013-4310.yml

```diff
@@ -119,10 +124,13 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
-  automated:
-  apache:
+  answer: |
+    The vulnerability was discovered by Vincent Danen, Director of Product Security at
+    RedHat. Vincent was looking into an announced chromium bug that hadn't yet been made
+    public and found this aswell. He started tracking the bugs status the same day.
```

Comment:
I found two errors in the grammar/spelling here. "aswell" should change to "as well" and "bugs" to "bug's".